### ISSUE TITLE
cc_inst dependancy on rgeos with geod=TRUE

### ISSUE THREAD

**Issue by wcornwell** [ropensci/CoordinateCleaner#93]
Sounds like a pretty big change to catch up with the changing spatial packages! Thanks for all your hard work on keeping this super useful package up-to-date!  

Anyway here is one bug I found since the change:
```
x <- data.frame(species = letters[1:10],
                decimalLongitude = c(runif(99, -180, 180), 37.577800),
                decimalLatitude = c(runif(99, -90,90), 55.710800))
cc_inst(x, value = "clean", buffer = 1, geod = TRUE)
```
returns the following error on my set up:

> Testing biodiversity institutions
> Error in explodePolygons(x, ...) : use sf or terra functions
> In addition: Warning message:
> In explodePolygons(x, ...) : No rgeos support in sp from October 2023;
> 

I think it works with `geod = FALSE`

**Comment by BrunoVilela** [comment]
Hi @wcornwell, I could not reproduce your error on my computer. Could you paste here the `r sessionInfo()`. Also I just uploaded some changes to the package on Github, they should fix some reported issues on version 3.0, it may be better if you test on this new version. Thanks!

**Comment by wcornwell** [comment]
I had an old versions of `geosphere` `rnaturalearth` installed and I updated `CoordinateCleaners`as you suggested.  Updating one of those packages seemed to fix the issue.  Thanks again!
